package.json

```diff
@@ -88,12 +90,11 @@
     "node-fetch": "^2.6.1",
     "preact": "10.15.1",
     "preact-custom-element": "^4.3.0",
+    "preact-iso": "2.2.0",
```

Comment:
Something in 2.3.0+ (I'm guessing https://github.com/preactjs/wmr/pull/864) results in `onLoadEnd` never being called on first page rendered, resulting in our loading bar going until the user switches pages.